communicator/ssh/communicator.go

```diff
@@ -236,7 +298,7 @@ func (c *comm) Download(path string, output io.Writer) error {
 		return nil
 	}
 
-	return c.scpSession("scp -vf "+strconv.Quote(path), scpFunc)
+	return c.scpSession("scp -vf "+path, scpFunc)
```

Comment:
I can check string for space and if i found it add quotes, if not - leave as is.